### ISSUE TITLE
Option to display binary name only

### ISSUE THREAD

**Issue by Zekario** [enkore/j4-dmenu-desktop#65]
I think it would be nice to have an option where only binaries with desktop files are shown (with the name of the binary only). Currently I only see a way to print the binary name in parentheses after the desktop file.

This would be useful to filter out all of the commandline-only programs that a user is unlikely to use dmenu to launch, while keeping the binary names.

**Comment by cdarken** [comment]
If I may add to this. It would also be useful to autocomplete a command that is not found in .desktop files, like shell files used to launch IDEs like Eclipse, Netbeans or Phpstorm. So, if I type `netbe <TAB>` autocomplete it to netbeans. I don't know if it's possible, but would be useful if it's easy to implement.

**Comment by enkore** [comment]
> I don't know if it's possible, but would be useful if it's easy to implement.

I don't think this is possible with normal dmenu, because we only get to send a list once of the available options and get a selection back.
